xbmc/Application.cpp

```diff
@@ -4301,7 +4301,9 @@ void CApplication::ProcessSlow()
 
   g_mediaManager.ProcessEvents();
 
-  CInputManager::Get().EnableRemoteControl();
+  if (CInputManager::Get().IsRemoteControlEnabled() &&
+      !CInputManager::Get().IsRemoteControlInitialized())
+    CInputManager::Get().EnableRemoteControl();
```

Comment:
mm was thinking about it but decided it wasn't worth adding an InitializeRemoteControl just for this, it wouldn't make any technical difference that I can think of but it would make more sense in this specific case.

Enable sets the enabled flag and then runs init.